src/main/java/org/example/notifications/RealEmailNotificationService.java

```diff
@@ -1,22 +1,24 @@
+package org.example.notifications;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
 import javax.mail.*;
 import javax.mail.internet.InternetAddress;
 import javax.mail.internet.MimeMessage;
 import java.util.Date;
 import java.util.Properties;
 
+@Component
 public class RealEmailNotificationService implements NotificationService {
-    private final String fromEmail;
-    private final String passwordKey;
-    private final String subjectMailMessage;
-    private final String emailNotificationText;
-
-    public RealEmailNotificationService(String fromEmail, String passwordKey, String subjectMailMessage,
-                                        String emailNotificationText) {
-        this.fromEmail = fromEmail;
-        this.passwordKey = passwordKey;
-        this.subjectMailMessage = subjectMailMessage;
-        this.emailNotificationText = emailNotificationText;
-    }
+    @Value("${fromEmail}")
```

Comment:
better would be refer to property via next path:
 email.sendFrom
 email.passwordKey
 email.notificationText
email.subject